src/pickers/range.tsx

```diff
@@ -1,10 +1,12 @@
-import compareAsc from 'date-fns/compare_asc';
-import isSameDay from 'date-fns/is_same_day';
+import compareAsc from 'date-fns/compareAsc';
+import isSameDay from 'date-fns/isSameDay';
 import React from 'react';
 import { DateFns, RangeDatePickerProps } from '../types';
 import BaseDatePicker from './base';
 import { composeEventHandlers, isInRange } from './utils';
 
+import { legacyParse } from '@date-fns/upgrade/v2';
```

Comment:
Can you move this import to line 3 just for convenience and code organization, please?

Reply:
Import line removed